src/meta/meta_bulk_load_service.cpp

```diff
@@ -797,6 +837,15 @@ void bulk_load_service::try_rollback_to_downloading(const std::string &app_name,
             "app({}) has been rollback to downloading for {} times, make bulk load process failed",
             app_name,
             _apps_rollback_count[pid.get_app_id()]);
+
+        bulk_load_result::type ret = bulk_load_result::BLR_INVALID;
+        if (get_app_ever_ingesting(pid.get_app_id())) {
+            ret = bulk_load_result::BLR_ERR_INGESTED_FAILED;
+        } else {
+            ret = bulk_load_result::BLR_ERR_FS_INTERNAL;    // TODO or BLR_ERR_CORRUPTION?
```

Comment:
It is not `BLR_ERR_FS_INTERNAL` nor `BLR_ERR_CORRUPTION`, just failed for too many rollback. I think bulk_load_result should add a new filed for it.

Reply:
Add a new result status: BLR_ERR_TRY_EXHAUST